docs/src/configuration/routes/proxy.md

```diff
@@ -0,0 +1,117 @@
+```
+
+## URL paths
+
+In the basic example above, the route preserves the URL path, `/foo`, in the request.
+
+If you want to proxy a route to `https://www.example.com` without the URL path `/foo`,
+add a trailing slash `/` to the `to` definition.
+​
+```yaml {location=".platform/routes.yaml"}
+https://{default}/foo:
+    type: proxy
+    to: https://www.example.com/
+```
+​
+The trailing slash makes the proxy route interpret the location as having a different path.
+So requests for `https://{default}/foo/index.html` are forwarded to `https://www.example.com/index.html`.
+
+To override the URL path entirely, define a route that contains its own path.
+For example:
+​
+```yaml {location=".platform/routes.yaml"}
+https://{default}/foo:
+    type: proxy
+    to: https://www.example.com/bar
+```
+​
+This route passes requests for `https://{default}/foo/index.html` to `https://www.example.com/bar/index.html`.
+​
+## Multiple apps with the same base URL
+​
+You can use proxy routes to map a single domain to multiple Platform.sh projects with their own subdomain/domain names.
+You might have a need to access multiple projects, each hosting specific applications for different languages.
+You want to serve them all at the same base URL with different paths
+(`https://example.com/en`, `https://example.com/fr`, and so on).
+
+Because domains can't be reused at Platform.sh, you can't just set the same domain for all projects.
+Use proxy routes so a single project can access different projects using the same base URL.
+​
+In the following example, a single project specifies proxy routes to three apps with the same `default` base URL.
+Each app handles a different language.
+​
+```yaml {location=".platform/routes.yaml"}
+https://{default}/en:
+    type: proxy
+    to: https://en.example.com/
+
+https://{default}/jp:
+    type: proxy
+    to: https://jp.example.com/
+
+https://{default}/pt:
+    type: proxy
+    to: https://pt.example.com/
+```
+​
+The apps behind the proxy need to ensure links to assets are shown to the target domain.
+For example, by changing `https://en.example.com/style.css` to `https://example.com/en/style.css`.
+
+{{< mermaid >}}
+sequenceDiagram
+    participant User
+    participant Project as Proxy project
+    participant En as En project
+    participant Jp as Jp project
+    participant Pt as Pt project
+    User->>+Project: example.com/en/index.html
+    Project->>+En: en.example.com/index.html
+    Note over En: Changes asset links
+    En-->>-Project: index.html
+    Project-->>-User: index.html
+    User->>+Project: example.com/jp/index.html
+    Project->>+Jp: jp.example.com/index.html
+    Note over Jp: Changes asset links
+    Jp-->>-Project: index.html
+    Project-->>-User: index.html
+    User->>+Project: example.com/pt/index.html
+    Project->>+Pt: pt.example.com/index.html
+    Note over Pt: Changes asset links
+    Pt-->>-Project: index.html
+    Project-->>-User: index.html
+{{< /mermaid >}}
+​
+### Large projects
```

Comment:
:hammer_and_wrench: :warning: Since this is a "caveat", the headline could be "caveat", or integrated into the presentation of the feature to make clear what the pros and cons of that feature are.

Reply:
"Caveat" was previous proposed as a heading and I changed it because I think it's too general to be very useful. I think it's much more useful to have a heading that tells what it applies to. If you have a small project, you can ignore this section. If you have a large project, you should read it and you'll learn the pros/cons for such projects. It's not a general piece of information, but something specific to specific use cases.